Cargo.toml

```diff
@@ -33,9 +33,9 @@ members = [
 ]
 
 [patch.crates-io]
-jsonrpsee = { git = "https://github.com/PureStake/jsonrpsee", branch = "moonbeam-v0.15" }
-jsonrpsee-core = { git = "https://github.com/PureStake/jsonrpsee", branch = "moonbeam-v0.15" }
-jsonrpsee-types = { git = "https://github.com/PureStake/jsonrpsee", branch = "moonbeam-v0.15" }
+jsonrpsee = { git = "https://github.com/PureStake/jsonrpsee", branch = "moonbeam-v0.15.1" }
+jsonrpsee-core = { git = "https://github.com/PureStake/jsonrpsee", branch = "moonbeam-v0.15.1" }
+jsonrpsee-types = { git = "https://github.com/PureStake/jsonrpsee", branch = "moonbeam-v0.15.1" }
```

Comment:
@librelois can you review (esp. the branch I created on `purestake/jsonrpsee`)?

I think this was necessary because our dependency graph included `v0.15.1` explicitly, so `v0.15.0` wasn't a match.